File: gluecodium/src/test/resources/smoke/field_constructors/output/android/com/example/smoke/FieldConstructorsPartialDefaults.java
/*

 *
 */

package com.example.smoke;

import android.support.annotation.NonNull;

public final class FieldConstructorsPartialDefaults {
    @NonNull
    public String stringField;
    public int intField;
    public boolean boolField;
    /**
     * <p>This is some field constructor with two parameters.
     * It is very important.
     * @param intField 
     * @param stringField 
     */
    public FieldConstructorsPartialDefaults(final int intField, @NonNull final String stringField) {
        this.intField = intField;
        this.stringField = stringField;
        this.boolField = true;
    }

    public FieldConstructorsPartialDefaults(final boolean boolField, final int intField, @NonNull final String stringField) {
        this.boolField = boolField;
        this.intField = intField;
        this.stringField = stringField;
    }


}

File: gluecodium/src/test/resources/smoke/field_constructors/output/android-kotlin/com/example/smoke/FieldConstructorsPartialDefaults.kt
/*

 *
 */

@file:JvmName("FieldConstructorsPartialDefaults")

package com.example.smoke


class FieldConstructorsPartialDefaults {
    @JvmField var stringField: String
    @JvmField var intField: Int
    @JvmField var boolField: Boolean




    /**
     * This is some field constructor with two parameters.
     * It is very important.
     * @param intField 
     * @param stringField 
     */
    constructor(intField: Int, stringField: String) {
        this.intField = intField
        this.stringField = stringField
        this.boolField = true
    }

    constructor(boolField: Boolean, intField: Int, stringField: String) {
        this.boolField = boolField
        this.intField = intField
        this.stringField = stringField
    }





}

